### PR TITLE
Merge branch-PPP-Marcus with Master

### LINKED ISSUE #6: Create individual Project Portfolio Pages

### PR DESCRIPTION
**Pull request by lfrostbytee**

### CODE REVIEW

File: docs/team/lfrostbytee.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: Ong Wee Marcus's Project Portfolio Page
+---
+
+### Project: Duke The Market
```
Comment (Benjamin-Sim) [inline docs/team/lfrostbytee.md:6]: Are we calling it Duke-The-Market?

Comment (lfrostbytee) [reply docs/team/lfrostbytee.md:6]: Let's settle on "Duke The Market" as we have previously submitted that as the name of our product. Our product name has also been set and can be seen on the [CS2103 website](https://nus-cs2103-ay2223s1.github.io/website/admin/teamList.html#cs2103-f09-2-duke-the-market)

Comment (Benjamin-Sim) [reply docs/team/lfrostbytee.md:6]: Sure, though there may be some places with the incorrect name "Duke-The-Market" which we need to change.
E.g. The [PR](https://github.com/nus-cs2103-AY2223S1/tp/pull/84) from our team repo to the upstream CS2103 repo.

Comment (lfrostbytee) [reply docs/team/lfrostbytee.md:6]: Yup, thanks for pointing that out. So far besides the PR, I haven't noticed anywhere else that uses "Duke-The-Market" yet. In the meantime, I have changed the title of the [PR](https://github.com/nus-cs2103-AY2223S1/tp/pull/84) to "Duke The Market"

**Review by Benjamin-Sim — APPROVED** [review]
LGTM! 👍

**Pull request merged**